tests/Database/DatabaseEloquentCollectionTest.php

```diff
@@ -202,14 +202,6 @@ public function testCollectionReturnsUniqueItems()
 	}
 
 
-	public function testLists()
```

Comment:
The test is superfluous.The eloquent collection does not have its own implementation.